Lib/fontTools/ttLib/tables/otTables.py

```diff
@@ -1297,6 +1298,144 @@ def preWrite(self, font):
 		return self.__dict__.copy()
 
 
+class ClipBox(getFormatSwitchingBaseTableClass("uint8")):
+
+	def as_tuple(self):
+		return tuple(getattr(self, conv.name) for conv in self.getConverters())
+
+	def __repr__(self):
+		return f"{self.__class__.__name__}{self.as_tuple()}"
+
+
+class ClipList(BaseTable):
+
+	def populateDefaults(self, propagator=None):
+		if not hasattr(self, "clips"):
+			self.clips = {}
+
+	def postRead(self, rawTable, font):
+		clips = {}
+		glyphOrder = font.getGlyphOrder()
+		for i, rec in enumerate(rawTable["ClipRecord"]):
+			rangesOverlap = False
+			for glyphID in range(rec.StartGlyphID, rec.EndGlyphID + 1):
+				try:
+					glyph = glyphOrder[glyphID]
+				except IndexError:
+					continue
```

Comment:
Shouldn't we warn about it? - seems sort of equally broken as overlap and we logged that

Reply:
i'm warning now